src/ast/ast.c

```diff
@@ -580,6 +580,26 @@ bool AST_ClauseContainsAggregation
 	return aggregated;
 }
 
+const char *AST_GetProjectionAlias
```

Comment:
Please copy the comments from the header file to this file.

Reply:
OK. Done in [23fa13d](https://github.com/RedisGraph/RedisGraph/pull/3107/commits/23fa13dd69f0528f3ca6b528f7b587ce54ba0569)